modules/serverless-logic-global-config-customization.adoc

```diff
@@ -0,0 +1,130 @@
+// Module included in the following assemblies:
+// * serverless-logic/serverless-logic-global-config-settings
+
+:_mod-docs-content-type: REFERENCE
+[id="serverless-logic-global-config-customization_{context}"]
+= Customization of global configurations
+
+After installing the {ServerlessLogicOperatorName}, you can access the `logic-operator-rhel8-controllers-config` ConfigMap file in the `openshift-serverless-logic` namespace. This configuration file defines how the operator behaves when it creates new resources in the cluster. However, changes to this configuration will not affect resources that already exist. 
+
+You can modify any of the options within the `controllers_cfg.yaml` key in the `ConfigMap`. 
+
+The following table outlines all the available global configuration options:
+
+[cols=3*,options="header"]
+|===
+|Configuration key
+|Default value
+|Description
+
+|`defaultPvcKanikoSize`
+|`1Gi`
+|The default size of Kaniko PVC when using the internal {ServerlessLogicOperatorName} builder manager.
+
+|`healthFailureThresholdDevMode`
+|`50`
+|How much time (in seconds) to wait for a devmode workflow to start. This information is used for the controller manager to create new devmode containers and setup the healthcheck probes.
+
+|`kanikoDefaultWarmerImageTag`
+|`gcr.io/kaniko-project/warmer:v1.9.0`
+|Default image used internally by the Operator Managed Kaniko builder to create the warmup pods.
+
+|`kanikoExecutorImageTag`
+|`gcr.io/kaniko-project/executor:v1.9.0`
+|Default image used internally by the Operator Managed Kaniko builder to create the executor pods.
+
+|`jobsServicePostgreSQLImageTag`
+|`registry.redhat.io/openshift-serverless-1/logic-jobs-service-postgresql-rhel8:{ServerlessProductVersion}`
+|The Jobs Service image for PostgreSQL to use, if empty, the {ServerlessLogicOperatorName} will use the default Apache Community one based on the current {ServerlessLogicOperatorName} version.
+
+|`jobsServiceEphemeralImageTag`
+|`registry.redhat.io/openshift-serverless-1/logic-jobs-service-ephemeral-rhel8:{ServerlessProductVersion}`
+|The Jobs Service image without persistence to use, if empty, the {ServerlessLogicOperatorName} will use the default Apache Community one based on the current {ServerlessLogicOperatorName} version.
+
+|`dataIndexPostgreSQLImageTag`
+|`registry.redhat.io/openshift-serverless-1/logic-data-index-postgresql-rhel8:{ServerlessProductVersion}`
+|The Data Index image for PostgreSQL to use, if empty, the {ServerlessLogicOperatorName} will use the default Apache Community one based on the current {ServerlessLogicOperatorName} version.
+
+|`dataIndexEphemeralImageTag`
+|`registry.redhat.io/openshift-serverless-1/logic-data-index-ephemeral-rhel8:{ServerlessProductVersion}`
+|The Data Index image without persistence to use, if empty, the {ServerlessLogicOperatorName} will use the default Apache Community one based on the current {ServerlessLogicOperatorName} version.
```

Comment:
```suggestion
|The data index image without persistence to use. If empty, the {ServerlessLogicOperatorName} uses the default Apache community image based on the current {ServerlessLogicOperatorName} version.
```

Reply:
Here "Data Index" is the name of the service so I will keep it as it is. Maybe I should add the word "service" to make it clearer. Rest is incorporated.